backend/de.metas.contracts/src/main/java/de/metas/contracts/interceptor/C_OrderLine.java

```diff
@@ -71,6 +77,24 @@ public void setSameFlatrateConditionsForWholeCompensationGroupWhenOneGroupLineCh
 		groupChangesHandler.recreateGroupOnOrderLineChanged(orderLine);
 	}
 
+	private boolean isOrderLineExcludedFromGroupFlatrateConditions(final I_C_OrderLine orderLine, final GroupId groupId)
+	{
+		if (groupId == null)
+		{
+			return true;
+		}
+
+		final boolean productExcludedFromFlatrateConditions = groupChangesHandler.
```

Comment:
simply: i guess u can return the result of isProductExcludedFromFlatrateConditions